package.json

```diff
@@ -9,7 +9,12 @@
   "main": "dist/src/index.js",
   "contributors": [],
   "scripts": {
-    "lint": "eslint --ext .ts .",
+    "lint": "npm run lint:eslint && npm run lint:prettier",
+    "lint:fix": "npm run lint:fix:eslint && npm run lint:fix:prettier",
+    "lint:eslint": "eslint --ext .ts .",
+    "lint:prettier": "prettier --check '{src,test}/**/*.ts'",
+    "lint:fix:eslint": "eslint --ext .ts,.js --fix ${ESLINT_OPTS} .",
+    "lint:fix:prettier": "prettier --write '{src,test}/**/*.ts'",
```

Comment:
This is great, but I have a terminology quibble: [Prettier is not a linter](https://prettier.io/docs/en/comparison.html).

This means that the `lint` script shouldn't also run Prettier, which messes with the seamless usability of your original scripts (i.e. people don't have to learn anything new, they can keep using `npm run lint` as before).

I think that since we're introducing a new tool, we shouldn't be afraid of adding a new set of scripts (especially because Prettier should be handled automatically by the IDE in most cases). If you disagree with this opinion, that's ok: my main, separate point is about Prettier not being a linter.

Proposed change:
```json
    "format": "prettier --check '{src,test}/**/*.ts'",
    "format:fix": "prettier --write '{src,test}/**/*.ts'",
    "lint": "eslint --ext .ts .",
    "lint:fix": "eslint --ext .ts,.js --fix ${ESLINT_OPTS} .",
```

Reply:
[Discussed on Slack](https://6river.slack.com/archives/C0AEZN7ME/p1582579150008500). Reached rough consensus on this being ok.